catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -123,6 +124,7 @@
         <argument>
             <reference interface="ddf.catalog.data.AttributeRegistry" availability="optional"/>
         </argument>
+        <argument ref="configurationApplication" />
```

Comment:
The `endpointUtil` takes the `configurationApplication` as an argument and the `configurationApplication` takes the `endpointUtil` as a property.  I think this is a CircularDependencyException

Reply:
Yeah i did it on the wrong class, fixed now tho

Reply:
Oh actually no, this one is the correct one.  I messed up and accidentally made the ConfigurationApplication take the EndpointUtil as a property but that was a mistake (meant to add to UserApplication). This one should be correct.